recipes/lapack95/meta.yaml

```diff
@@ -0,0 +1,38 @@
+{% set name = "lapack95" %}
+{% set version = "3.0.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: http://www.netlib.org/lapack95/lapack95.tgz
+  sha256: 55711e5b1c5ddc9c7dd1d3ea15c37fc5b400783ac6ea573a75e2b1358c2c479b
+
+build:
+  number: 0
+  skip: true  # [win]
+
+requirements:
+  build:
+    - {{ compiler('fortran') }}
+  host:
+    - libblas
+    - liblapack
+
+test:
+  commands:
+    - test -f $PREFIX/lib/liblapack95.a
```

Comment:
```suggestion
    - test -f $PREFIX/lib/liblapack95$SHLIB_EXT
```

Reply:
Also the static library is not being produced right? Might be worth adding a test that ensure the static library is not packaged

Reply:
@jakirkham the static library is being produced and packaged.

Reply:
I mentioned here https://github.com/conda-forge/staged-recipes/pull/14212#discussion_r592581467 that the compiling of the static library is hardcoded on the makefile for this package. As requested by @isuruf I changed to use the shared libraries as dependencies but I am not comfortable with rewriting the makefile for this library to have a shared library at least not at this moment if that would be ok.